test/karma.conf.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (config) {
             'app/lib/command.js',
             'app/lib/condition.js',
             'app/lib/manifest-reader.js',
+            'app/lib/boolean-phrase-parser.js',
```

Comment:
There should be a similar line in Gruntfile.js. I know it sucks but I didn't find a way to remove this annoying duplication yet. This happens because the files need to be included in specific order.